Gemfile.lock

```diff
@@ -228,9 +226,6 @@ GEM
       net-protocol
       timeout
     nio4r (2.5.8)
-    nokogiri (1.13.3)
-      mini_portile2 (~> 2.8.0)
-      racc (~> 1.4)
     nokogiri (1.13.3-x86_64-linux)
```

Comment:
this fights back for me on a local env (mac) - I'm going to submit a commit that brings this back to the general instance

Reply:
nice catch!